### PR TITLE
Wire-up the payment plans so users can switch.

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner @camhashemi

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/teamSettings/paymentPlan.js [hunk 1]
```diff
@@ -46,34 +43,103 @@ angular.module('kifi')
...
+      name: currentPlan.name,
+      cycle: currentPlan.cycle //month
+    };
+
+    $scope.isPaidPlanName = function (planName) {
+      return planName && planName.toLowerCase().indexOf('free') === -1;
+    };
+
+    $scope.isFreePlanName = function (planName) {
+      return planName && planName.toLowerCase().indexOf('free') > -1;
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/paymentPlan.js:86]: I think the goal is to get away from this, maybe I should add `isPaid` to the API @andrewconner ?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/paymentPlan.js:86]: agreed, I will implement the changes here and on server in another PR.

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/paymentPlan.js:86]: kk I'll merge now and add an update later.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/paymentPlan.js:86]: Semi-sadly, seems like we need to refactor it to be `isFree`, since you can have an account where price per cycler per user = 0 and `isPaid = true`, for our purposes. That will be in the separate PR @andrewconner @cvializ
